sdks/android/app/src/main/c/runtime-bootstrap.c

```diff
@@ -413,6 +416,9 @@ Java_org_mono_android_AndroidRunner_runTests (JNIEnv* env, jobject thiz, jstring
 	setenv ("MONO_LOG_MASK", "all", 1);
 	// setenv ("MONO_VERBOSE_METHOD", "GetCallingAssembly", 1);
 
+	/* uncomment to enable interpreter */
+	// mono_jit_set_aot_mode (1000 /* MONO_EE_MODE_INTERP */);
```

Comment:
`MonoEEMode` is defined in `mini-runtime.h` which is not part of the SDK, can we either move the enum declaration to a header that's part of the SDKs or distribute `mini-runtime.h` with the SDK

Reply:
Yes, we need to move that to a public header.  For now you can work around the way as suggested here (just use the integer constant).

iOS uses a slightly different API which isn't suitable for Android (it's tailored for the use in an FullAOT environment).